managers/kafka/interface.py

```diff
@@ -0,0 +1,20 @@
+""" An abstract base class used for abstracting the contents of the Kafka domain away from main """
+
+from abc import ABC, abstractmethod
+from asyncio import AbstractEventLoop
+
+
+class KafkaManager(ABC):
```

Comment:
This NApp is supposed to be a plugin/adapter for Kafka, tightly coupled with Kafka as a broker. No need for this layer of indirection here with this forced interface. It it ever needs another broker sender/manager it'll be a different NApp. Less is more based on the design decisions we've made, also code is read more times than it's written typically. The serializer module and suite is acceptable though, since we might change the serialization approach one day, but still also no need to still force a layer indirection as of now.